cpp/include/cudf/detail/utilities/cuda.cuh

```diff
@@ -103,6 +104,43 @@ __device__ T single_lane_block_sum_reduce(T lane_value) {
   return result;
 }
 
+/**
+ * @brief for each warp in the block do a reduction (summation) of the
+ * `__popc(bit_mask)` on a certain lane (default is lane 0).
+ * @param[in] bit_mask The bit_mask to be reduced.
+ * @return[out] result of each block is returned in thread 0.
+ */
+template <class bit_container, int lane = 0>
+__device__ __inline__ size_type single_lane_popc_block_reduce(bit_container bit_mask) {
```

Comment:
Can you rename this function `single_lane_block_popc_reduce` to match `single_lane_block_sum_reduce`?

Reply:
Done